github/filecoin-project.yml

```diff
@@ -4059,18 +3926,11 @@ repositories:
         - proofs-crate-owners
       maintain:
```

Comment:
On all other repos, `proofs` has only push access. I don't think `merkletree` should be a special case here.